src/landpatterns/BGA/package.stanza

```diff
@@ -124,9 +233,19 @@ public defn BGA (
     density-level
   )
 
+doc: \<DOC>
+@title Get BGA Planner
+@desc This function returns the planner for the BGA package.
```

Comment:
This is more a conversion function - ie, it converts the PadPlanner type from the `Package` definition into a `BGA-PadPlanner`.